readme.txt

```diff
@@ -65,6 +65,10 @@ Web Stories are for everyone! If you are a site owner, content creator, or publi
 3. Explore our WYSIWYG editor with easy access to many editing tools (shown here: zooming and cropping media)
 4. Get started building your first story by editing one of our expressive, fully built template stories included with the editor.
 
+== Terms of Service ==
+
+By using this plugin, you agree to [Google's Terms of Service](https://policies.google.com/terms). By using third-party imagery and video provided by [Unsplash](https://unsplash.com/terms) and [Coverr](https://coverr.co/license), you agree to adhere to their ToS and licensing terms."
```

Comment:
```suggestion
By using this plugin, you agree to [Google's Terms of Service](https://policies.google.com/terms). By using third-party imagery and video provided by [Unsplash](https://unsplash.com/terms), [Coverr](https://coverr.co/license) and [Tenor](https://tenor.com/legal-terms), you agree to adhere to the respective Terms of Service.
```

A suggestion to include @swissspidy 's comment. Also, spelling out ToS as Terms of Service as I don't think it's defined anywhere.